content/unexpected-reboots.md

```diff
@@ -0,0 +1,441 @@
+```
+
+The first command will make sure your package lists are up to date.
+The second will install the application memtester.
+
+2. Once memtester is installed, instruct it to test **most** of the RAM, leaving enough for the OS to continue running.
+
+For example, if your system has 16GB of RAM, test 12GB or 14GB.
+
+To start the test, run these `Terminal` commands:
+
+```bash
+sudo memtester 12GB 3 > ~/memtester.txt
+```
+
+This command will run memtester and test 12GB of RAM 3 times, then save the output to a file called "memtester.txt" in your Home folder.
+
+> **IMPORTANT NOTE:** Close all applications other than Terminal when you run memtester. This will likely take a few hours (the application runs multiple passes over the memory) so it would be best to run it overnight.
+
+
+### Thermals
+
+Modern hardware is designed to shut systems down when they reach temperatures that may be damaging to the internal components. Typically these thresholds are in the upper 80s or 90s Celsius, depending on hardware.
+
+If your system is spontaneously shutting down, this may be caused by overheating. Systems with dedicated GPUs tend to run hot under normal circumstances, so noticing an overheating problem can be challenging from ambient temperature alone. 
+
+The temperatures of your CPU cores and GPU card can be checked through software.
+
+#### Run `lm-sensors`
+
+`lm-sensors` (installed by default on Pop!\_OS) is a text-based tool that runs in a `Terminal`.
+
+1. Install `lm-sensors` (If not installed)
+
+Open a `Terminal` with <kbd>SUPER</kbd>+<kbd>T</kbd> (Pop!\_OS) or <kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>T</kbd> (Ubuntu) run the following commands:
+
+```bash
+sudo apt update
+sudo apt install lm-sensors
+```
+
+2. Get Sensor Output
+
+```bash
+sensors
+```
+
+This command will generate output like this:
+
+```
+coretemp-isa-0000
+Adapter: ISA adapter
+Package id 0:  +55.0°C  (high = +100.0°C, crit = +100.0°C)
+Core 0:        +53.0°C  (high = +100.0°C, crit = +100.0°C)
+Core 1:        +53.0°C  (high = +100.0°C, crit = +100.0°C)
+Core 2:        +53.0°C  (high = +100.0°C, crit = +100.0°C)
+Core 3:        +53.0°C  (high = +100.0°C, crit = +100.0°C)
+Core 4:        +55.0°C  (high = +100.0°C, crit = +100.0°C)
+Core 5:        +53.0°C  (high = +100.0°C, crit = +100.0°C)
+Core 6:        +52.0°C  (high = +100.0°C, crit = +100.0°C)
+Core 7:        +54.0°C  (high = +100.0°C, crit = +100.0°C)
+
+system76_acpi-acpi-0
+Adapter: ACPI interface
+CPU fan:        0 RPM
+GPU fan:        0 RPM
+CPU temp:     +55.0°C  
+GPU temp:     +54.0°C  
+
+iwlwifi_1-virtual-0
+Adapter: Virtual device
+temp1:        +69.0°C  
+
+pch_cometlake-virtual-0
+Adapter: Virtual device
+temp1:        +60.0°C  
+
+BAT0-acpi-0
+Adapter: ACPI interface
+in0:          12.95 V  
+curr1:         0.00 A  
+```
+
+#### Psensor
+
+If you prefer a GUI tool which provides graphing over time. The application Psensor can be installed from the Pop!\_Shop, or through the `Terminal` with this command:
```

Comment:
The first sentence here is incomplete.

Reply:
I split the sentence into a fragment and a sentence instead of a longer, compound sentence. Thanks for catching that.